### DISCUSSION TITLE
Build Fails for darwin_arm64 on Mac

### DISCUSSION THREAD

**Discussion by jiaocoll** [goreleaser/goreleaser#3294] [Q&A] [answered]
my computer system version:
![image](https://user-images.githubusercontent.com/62785738/183293723-7536b935-1210-474b-94dd-a20fe4858f40.png)

GO version:
![image](https://user-images.githubusercontent.com/62785738/183293761-5ecf0321-dc6e-4d6b-9472-f0ecad0b998f.png)

goreleaser version:
![image](https://user-images.githubusercontent.com/62785738/183293784-fcc55716-cf2c-4dd4-8aca-b01da30c8435.png)

goreleaser check:
![image](https://user-images.githubusercontent.com/62785738/183293820-abc5d9d4-ffb4-4603-aee3-f8b7f5c7aae9.png)

my go version is 1.17.5 > 1.16. so I don't konw why i don't get drawin_arm64.

**Comment by caarlos0** [comment, marked as answer]
old goreleaser version, and probably you have multiple go versions installed and goreleaser is picking up the wrong one.

either way, I recommend updating goreleaser.

**Reply by jiaocoll** [reply]
Thank you very much！
It may indeed be because my goreleaser version is too low.
After updating the gorelease version it did return to normal.
